.husky/pre-commit

```diff
@@ -1,6 +1,6 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-. "$(dirname "$0")/common.sh"
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
 
-yarn lint-staged
-yarn lingui
\ No newline at end of file
+# todo @l03tj3, enable after pr merge > 263-goodswap-should...
+# yarn lingui
+# yarn lint-staged
```

Comment:
not run on last commit because of eslint fixes which are already done on other pr. 
will fix everything that's left there after merge